session/src/test/java/uk/gov/di/ipv/cri/common/api/service/SignedJWTBuilder.java

```diff
@@ -142,8 +149,12 @@ SignedJWT build() {
             if (includeSubject) {
                 jwtClaimSetBuilder.subject(ipv_session_id);
             }
-            if (includeSharedClaims) {
+            if (Objects.isNull(sharedClaims) && includeSharedClaims) {
```

Comment:
I'm generally opposed to using the Objects class for predicates like this since it's meant for lambda filtering e.g. `list.filter(Objects::isNull)`, the better approach would have been `sharedClaims == null` but this is fine for what we need.

Reply:
Yeah, it just a pattern we being using Tal was a fan is this, so I don't really have any opinion about this. it the same code under the hood.